huaweicloud/resource_huaweicloud_cdn_domain.go

```diff
@@ -77,6 +244,74 @@ func resourceCdnDomainV1() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+			"configs": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"origin_protocol": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								"follow", "http", "https",
+							}, false),
+						},
+						"https_settings":             &httpsConfig,
+						"retrieval_request_header":   &requestAndResponseHeader,
+						"http_response_header":       &requestAndResponseHeader,
+						"url_signing":                &authOpts,
+						"force_redirect":             &forceRedirectAndCompress,
+						"compress":                   &forceRedirectAndCompress,
+						"cache_url_parameter_filter": &cacheUrlParameterFilter,
+					},
+				},
+			},
+
+			"cache_settings": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"follow_origin": {
+							Type:     schema.TypeBool,
+							Optional: true,
+						},
+						"rules": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"rule_type": {
+										Type:     schema.TypeInt,
+										Required: true,
+									},
+									"content": {
+										Type:     schema.TypeString,
+										Optional: true,
```

Comment:
If the `content` can be updated to a null value, we should add the `Compated:true` label.
If so, pls check other parameters.

Reply:
fixed